scripts/exregional_make_ics.sh

```diff
@@ -359,6 +361,7 @@ fi
 #
 thomp_mp_climo_file=""
 if [ "${EXTRN_MDL_NAME_ICS}" != "HRRR" -a \
+     "${EXTRN_MDL_NAME_ICS}" != "HRRRDAS" -a \
      "${EXTRN_MDL_NAME_ICS}" != "RAP" ] && \
    [ "${SDF_USES_THOMPSON_MP}" = "TRUE" ]; then
   thomp_mp_climo_file="${THOMPSON_MP_CLIMO_FP}"
```

Comment:
RRFSE is NOT using climo data for the microphysics, correct?  Sorry if this question doesn't make sense here, just asking because this code seems to set the climo file.

Reply:
I am setting RRFSE with HRRRDAS ICs to use whatever HRRR ICs use - not to use climo, I think.